tests/e2e/api/controllers/bulk-get.spec.js

```diff
@@ -208,9 +218,16 @@ describe('bulk-get handler', () => {
         return utils.requestOnTestDb(offlineRequestOptions);
       })
       .then(result => {
-        expect(result.results.length).toEqual(2);
-        expect(result.results[0].id).toEqual('allowed_task');
-        expect(result.results[1].id).toEqual('allowed_target');
+        chai.expect(result.results).excludingEvery('_rev').to.deep.equal([
```

Comment:
I learnt something!

Reply:
The absolutely beautiful part is that the properties are excluded from both LHS and RHS compared items.